CMakeLists.txt

```diff
@@ -608,6 +608,7 @@ if(NOT GPU_ARCHS AND USER_GPU_TARGETS)
         PACKAGE_NAME examples
    )
    add_subdirectory(example)
+   add_subdirectory(tile_engine)
```

Comment:
can we conditionally include this folder? today ck cmake really takes a long time to cmake

Reply:
It takes so, because of code generation in FA ;P